packages/app/src/components/challenges/OutcomeCard.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { ReactElement, useRef, useState } from 'react';
+import {
+    Text,
+    View,
+    StyleSheet,
+    TextInput,
+    TouchableOpacity,
+    KeyboardAvoidingView,
+    Platform
+} from 'react-native';
+import { Card, colors, Notch, sizes, Title } from '../UI';
+import { FormType } from './CreateChallengeScreen';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import { ScrollView } from 'react-native-gesture-handler';
+
+type Props = {};
```

Comment:
Can you add typing here

Reply:
This seems to be from an older version of the file.